content/en/sip-print.md

```diff
@@ -45,3 +45,24 @@ But what you find are simplistic PC-based handset tools or expensive choices tha
 SIP Print call recording is just like your phone system: easy to install, easy to operate and easy to afford. Three versions are available, allowing for efficient scalability. All offer enterprise-class call recording performance for training, Q/A, compliance or regulatory purposes at the corporate, branch or small office level.
 
 With the ability to record advanced functions like voicemail and “follow-me” calls forwarded to mobile phones or off-premise phone numbers, SIP Print shows the expandability and value of IP standards.
+
+CALL RECORDING / PLAYBACK FEATURES
```

Comment:
@yasuyoshiakiyama when you copy paste in, you have to re-format to markdown format. I'll do the English but please fix it in the Japanese.